kaldb/src/main/java/com/slack/kaldb/logstore/schema/SchemaAwareLogDocumentBuilderImpl.java

```diff
@@ -348,30 +135,29 @@ public enum FieldConflictPolicy {
     CONVERT_AND_DUPLICATE_FIELD
   }
 
-  private static final Map<FieldType, FieldDef> defaultPropDescriptionForType =
+  private static final String DUMMY_FIELD = "DUMMY_FIELD";
```

Comment:
This needs a comment explaining why a dummy field is being used here for the default properties.